templates/internet-of-things/gateways.html

```diff
@@ -1,387 +1,657 @@
+        </p>
+      </div>
     </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <p class="p-heading--3">Container-friendly</p>
-  </div>
-
-  <div class="row u-equal-height p-divider">
-    <div class="col-4 p-divider__block">
-      <h4>Snaps</h4>
-      <p>Snaps facilitate system integration thanks to their portability and reusability. Transactionally updated packages enable cost-effective maintenance for complex installations.</p>
+    <div class="row">
+      <div class="col-9 col-medium-6 col-start-large-4">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Largest pool of talent</h3>
+          </div>
+          <div class="col-6 col-medium-3 p-section--shallow">
+            <p>
+              Developers love Ubuntu. By adopting the most popular Linux distribution, you can leverage the strong community of developers, vendors and system integrators familiar Ubuntu.
+            </p>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Open source software</h3>
+          </div>
+          <div class="col-6 col-medium-3 p-section--shallow">
+            <p>
+              Complex software is increasingly collaborative and open source. This was our philosophy from the beginning. Harness the power of our open source software and build on top of contributions from the community. Build on your own, or seek our help. We will support you.
+            </p>
+          </div>
+        </div>
+      </div>
     </div>
-    <div class="col-4 p-divider__block">
-      <h4>Docker</h4>
-      <p>Docker containers complement snap packages to deliver full flexibility in the development and deployment of applications at the edge.</p>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>
+          Gateways
+          <br class="u-hide--small" />
+          for new business models
+        </h2>
+      </div>
+      <div class="col">
+        <p>Build an ecosystem around your hardware: partner with developers, vendors and system integrator.</p>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="https://www.brighttalk.com/webcast/6793/290087">Watch the app store for Linux webinar&nbsp;&rsaquo;</a>
+        </p>
+      </div>
     </div>
-    <div class="col-4 p-divider__block">
-      <h4>Kubernetes</h4>
-      <p>Single node Kubernetes packaged in a snap, can be deployed to edge gateways in order to orchestrate containers running a variety of IoT services.</p>
+    <div class="row">
+      <div class="col-6 col-medium-6 col-start-large-7">
+        <div class="p-equal-height-row-2">
```

Comment:
Not sure that this will work since this isn't actually a `p-equal-height-row`, but the hr should be the equivalent of setting the `has-divider-3` on the real equal-row component where the hr above the links spans the whole 6 cols
![image](https://github.com/user-attachments/assets/c8605ea9-006d-4f81-9412-10f10b488e2e)